packages/react-components/react-toast/src/state/useToaster.ts

```diff
@@ -38,14 +38,16 @@ export function useToaster<TElement extends HTMLElement = HTMLDivElement>(option
   }, []);
 
   const pauseAllToasts = React.useCallback(() => {
-    for (const toastId of toaster.visibleToasts) {
+    const visibleToasts = Array.from(toaster.visibleToasts.values());
+    for (const toastId of visibleToasts) {
       const toast = toaster.toasts.get(toastId);
       toast?.imperativeRef.current?.pause();
     }
   }, [toaster]);
 
   const playAllToasts = React.useCallback(() => {
-    for (const toastId of toaster.visibleToasts) {
+    const visibleToasts = Array.from(toaster.visibleToasts.values());
```

Comment:
had to make this change because `@fluentui/react-portal-compat` has ES5 as a target and there were TS compilation issues with interating sets/iterators directly